charts/jenkins/values.yaml

```
@@ -628,6 +628,8 @@ agent:
   kubernetesReadTimeout: 15
   maxRequestsPerHostStr: "32"
   namespace:
+  # private registry for agent image
+  jnlpregistry:
```

This should be commented out if not used, see examples further down in the file
```suggestion
  # jnlpregistry:
```

---

if I look up, there are examples as mine
```
  # URL for connecting to the Jenkins controller
  jenkinsUrl:
  # connect to the specified host and port, instead of connecting directly to the Jenkins controller
  jenkinsTunnel:
```
I wanted to be consistent with it, if I need to change it to get merge, let me know it is not big thing